src/Templates/cs/Templates/Templates/CSTypeEditor/CustomControlLibrary.Protocol/GlobalUtilities/GlobalUtilities.cs

```diff
@@ -0,0 +1,190 @@
+﻿// -------------------------------------------------------------------
```

Comment:
Do we really need this header in the template? Are the developers allowed to modify this file?

Reply:
I took it from the Designer, and the Designer took it from the runtime, if I remember correctly.
The header is the same as in the designer.
It's MIT so, it should be OK.
Let me check again.

Reply:
I'll also check with @RussKie.